Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16-alpine
+
+WORKDIR /grid_proxy_server
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN go build -o server cmds/proxy_server/main.go
```

Comment:
that should be a multistage build docker file

Reply:
resolved here: https://github.com/threefoldtech/grid_proxy_server/pull/10/files